src/main/native/cpp/analysis/FilteringUtils.cpp

```diff
@@ -91,3 +91,44 @@ units::second_t sysid::GetMeanTimeDelta(const Storage& data) {
 
   return std::accumulate(dts.begin(), dts.end(), 0_s) / dts.size();
 }
+
+void sysid::TrimQuasistaticData(std::vector<PreparedData>* data,
+                                double motionThreshold) {
+  data->erase(std::remove_if(data->begin(), data->end(),
+                             [motionThreshold](const auto& pt) {
+                               return std::abs(pt.voltage) <= 0 ||
+                                      std::abs(pt.velocity) < motionThreshold;
+                             }),
+              data->end());
+}
+
+void sysid::ApplyMedianFilter(std::vector<PreparedData>* data, int window) {
+  size_t step = window / 2;
+  frc::MedianFilter<double> medianFilter(window);
```

Comment:
Since the MedianFilter is used in a .cpp file, add `#include <frc/MedianFilter.h>` to the top and remove it from FilteringUtils.h.